src/OpenLoco/Input/MouseInput.cpp

```diff
@@ -648,11 +649,13 @@ namespace OpenLoco::Input
     // 0x004BB116 TODO: Move to a better file
     static void treeInteract(Map::TreeElement* tree, const Map::Pos2 pos)
     {
-        registers regs{};
-        regs.edx = reinterpret_cast<uint32_t>(tree);
-        regs.ax = pos.x;
-        regs.cx = pos.y;
-        call(0x004BB116, regs);
+        GameCommands::setErrorTitle(StringIds::error_cant_remove_this);
+        GameCommands::TreeRemovalArgs args;
+        args.pos = pos;
+        args.baseZ = tree->baseZ();
+        args.elementType = tree->rawData()[0];
```

Comment:
annoyingly i dont have enough information to accurately split this up into its component parts atm. I think only rotation (first 3 bits) is unique but as I am unsure the whole byte is being sent.